### DISCUSSION TITLE
Why does the repo lack a `package-lock.json`?

### DISCUSSION THREAD

**Discussion by srid** [saadeghi/daisyui#2217] [Q&A] [answered]
Could someone explain why `package-lock.json` is missing in the repo? This would make packaging daisyui in Nix a lot simpler. https://github.com/NixOS/nixpkgs/issues/247043

**Comment by saadeghi** [comment, marked as answer]
A package-lock.json file [will be ignored by NPM](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#files) anyway:

>some files are always ignored:
> - ...
> - package-lock.json


Almost no package on npm have a lock file included. See Tailwind CSS for example: https://unpkg.com/browse/tailwindcss@3.3.3/
